pallets/capacity/src/lib.rs

```diff
@@ -100,6 +101,16 @@ pub mod pallet {
 	};
 	use sp_runtime::traits::{AtLeast32BitUnsigned, MaybeDisplay};
 
+	/// A reason for freezing funds.
+	#[pallet::composite_enum]
+	pub enum FreezeReason {
```

Comment:
FreezeReason is required for the fungible traits. The enum is defined and used by capacity and time-release. Comments in the Parity deprecation pr indicated that there should only be one enum: 

> Both Holds and Freezes require an identifier which is configurable and is expected to be an enum aggregated across all pallet instances of the runtime.

This enum is scoped to each pallet, but we may want to move it to a common location within the runtime, if possible.